components/tyk-dashboard/values.yaml

```diff
@@ -0,0 +1,151 @@
+# Determines whether or not the dashboard component should be installed.
+enabled: true
```

Comment:
do we need this `enabled` in component chart? if users install this chart, i assume it should be enabled?

Reply:
Fixed